node/tests/setup.js

```diff
@@ -2,6 +2,6 @@ import { beforeAll } from "@jest/globals";
 import { Logger } from "..";
 
 beforeAll(() => {
-    Logger.init("error");
-    Logger.setLoggerConfig("error");
+    Logger.init("off");
```

Comment:
Is it still too noisy with `error` level? If yes, update this for java too (`TestConfiguration.java`)

Reply:
Less noisy, but still irrelevant noise, the problem is that we get used to seeing errors and warns, so we dismiss it, regardless of what is it. Until we create a better platform, I prefer having warns and error that when we see, we act about.
I will.